code/modules/mob/living/say.dm

```diff
@@ -94,19 +94,58 @@ var/list/channel_to_radio_key = new
 		if(S.speaking && (S.speaking.flags & NO_STUTTER || S.speaking.flags & SIGNLANG))
 			continue
 
-		if((HULK in mutations) && health >= 25 && length(S.message))
+		var/force_slurring = 0
+		var/force_shouting = 0
+		var/force_stuttering = 0
+		var/force_mute = 0
+
+		for(var/datum/modifier/M in modifiers)
+			if(M.speech_slurring)
+				force_slurring += M.speech_slurring

```

Comment:
So if I have two modifiers that want to make me _not_ slur, but only one that _does_, I won't slur? Is that the pattern we want to go for here?

Reply:
I can't quite imagine another way to do it that wouldn't be infuriating (influencing RNG)